spec.bs

```diff
@@ -416,21 +430,44 @@ a [=batching scope=] |batchingScope|, an [=origin=] |reportingOrigin| and a
             empty|empty=] [=list=].
     1. [=list/Append=] |entry|'s [=contribution cache entry/contribution=] to
         |batchedContributions|[|debugDetails|].
+1. If |batchedContributions| [=map/is empty=]:
+    1. Let |debugDetails| be a new [=debug details=] with the following items:
+        : [=debug details/enabled=]
```

Comment:
Do we need to set these default values explicitly?

Reply:
No, removed